java/code/src/com/redhat/rhn/frontend/xmlrpc/errata/ErrataHandler.java

```diff
@@ -1132,6 +1139,8 @@ private Object getRequiredAttribute(Map map, String attribute) {
      *          #prop_desc("string", "advisory_type", "Type of advisory (one of the
      *                  following: 'Security Advisory', 'Product Enhancement Advisory',
      *                  or 'Bug Fix Advisory'")
+     *          #prop_desc("string", "advisory_status", "Status of advisory (one of the
+     *                  following: 'final', 'testing', or 'stable'")
```

Comment:
`retracted` is missing here. But do we want this here?

Reply:
Added.